analysis/wwz/get_wwz_yields.py

```diff
@@ -935,9 +935,9 @@ def main():
 
     # Make plots
     if args.make_plots:
-        make_plots(histo_dict,sample_dict_mc,sample_dict_data,save_dir_path=out_path,apply_nsf_to_cr=False)
```

Comment:
Let's leave the standard plotting script the one tunred on by default

Reply:
Done